README.md

```diff
@@ -6,7 +6,7 @@ Shortcuts for wasm
 
 | wah       | wasm                                      |
 |---------------|-------------------------------------------|
-| `(0 = 1)`     | `(set_local 0 (f64.const 0))`             |
+| `(0 = 1)`     | `(set_local 0 (f64.const 1))`             |
```

Comment:
It's actually `(set_local 0 (i32.const 1))` as wah will only use f64 for literals with a decimal.